synapse/cortex.py

```diff
@@ -44,6 +45,7 @@ def __init__(self, name, msg):
     'ram':synapse.cores.ram.initRamCortex,
     'sqlite':synapse.cores.sqlite.Cortex,
     'postgres':synapse.cores.postgres.Cortex,
+    'sortedram':synapse.cores.sortedram.initSortedRamCortex,
```

Comment:
I'd potentially be all for having this live within the constructor for ram://, possibly with the addition of sorted=1 in the url parameters?